dom.bs

```diff
@@ -1113,6 +1121,11 @@ participate in a tree structure.</p>
  <a for="event listener">callback</a>, and <a for="event listener">capture</a> is
  <var>listener</var>'s <a for="event listener">capture</a>, then <a for=list>append</a>
  <var>listener</var> to <var>eventTarget</var>'s <a>event listener list</a>.
+
+  <li><p>If <var>signal</var> is not null then <a for=AbortSignal lt=add>add the following</a>
+  abort steps to it:
+    <li><a for=list>remove</a> <var>listener</var> from <var>eventTarget</var>'s
+    <a for=EventTarget>event listener list</a>.
```

Comment:
What if the signal is already aborted? Probably the whole algorithm should no-op?